<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgQTreeWidget</class>
 <widget class="QDialog" name="dlgQTreeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Demo del Control QTreeWidget</string>
  </property>
  <widget class="QLabel" name="lblRegistro">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Total de Registros</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTotal">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTreeWidget" name="tvwData">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>961</width>
     <height>721</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
